### PR TITLE
PERF-3303 Document how to configure encryption in Genny workloads

### PR DESCRIPTION
**Pull request by erwee**

### CODE REVIEW

File: docs/using.md [hunk 1]
```diff
@@ -833,6 +834,58 @@ This will create new Actor .cpp and .h files, an example workload yaml, as well
...
+      pii.ssn: { type: "string", algorithm: "deterministic" }
+  - Database: encrypted_db
+    Collection: qe_encrypted_coll
+    EncryptionType: qe
+    QueryableEncryptedFields:
+      name: { type: "string", queries: [{queryType: "equality"}] }
+      pii.ssn: { type: "string", queries: [{queryType: "equality"}] }
+```
+
+The required `Database` and `Collection` fields specify the namespace of the encrypted collection. Each node in this sequence must have a unique combination of these two names in order to avoid two different encryption schemas for the same namespace on the same URI.
```
Comment (jimoleary) [inline docs/using.md:860]: Should we raise a feature request to have genny enforce this?

**Review by jimoleary — COMMENTED** [review]
The text looks good, I have some questions about follow on tickets.

File: docs/using.md (hunk 1 above)
Comment (erwee) [reply docs/using.md:860]: This is already enforced in [here](https://github.com/mongodb/genny/blob/06c2e78ae75850fc9bc73922a87e650a88bfb84d/src/gennylib/src/encryption.cpp#L617:L623)

File: docs/using.md [hunk 2]
```diff
@@ -833,6 +834,58 @@ This will create new Actor .cpp and .h files, an example workload yaml, as well
...
+
+During client pool setup, key vaults in each unique URI are dropped & created once, when the first client pool for that URI is created. Data keys for encrypted namespaces are generated only once, if the associated key vault & URI does not yet contain keys for that namespace. This means that if two client pools have a similar URI and key vault namespace, then the encrypted collections they have in common will be using the same data keys.
+
+An Actor that wishes to perform encrypted operations on an encrypted collection must be using a client pool with an `EncryptedCollections` subnode that includes the collection's
+namespace.
+
+By default, encrypted CRUD operations require a `mongocryptd` daemon to be running and listening on localhost:27020. Genny will **not** automatically spawn this daemon on a workload run. Alternatively, one can set `Encryption.UseCryptSharedLib` to `true`, and provide the path to the `mongo_crypt_v1.so` shared library file using `Encryption.CryptSharedLibPath`.
```
[the remark this answers is not in the document]
Comment (erwee) [reply docs/using.md:885]: Not sure what you mean by auto genny workloads test_control. Could you elaborate?

[the remark this answers is not in the document]
Comment (jimoleary) [reply docs/using.md:885]: This is nothing to do with your PR, I'm just wondering if we can parameterize 
[test_control.auto_genny_workload.yml](https://github.com/10gen/dsi/blob/master/configurations/test_control/test_control.auto_genny_workload.yml)  to start mongocrytpd if a binary is set or available. For example, something like [tsbs_load](https://github.com/10gen/dsi/blob/ae9cab4a2a80ba194f9abad7b0762689d62f62d9/configurations/test_control/test_control.tsbs_load.yml#L124-L126).

**Review by jimoleary — APPROVED** [review]
LGTM on this, but a question on whether we want to create a ticket to support mongocryptd else where.

File: docs/using.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (erwee) [reply docs/using.md:885]: Of course! AutoRun will not work if these workloads are configured to use the daemon. Filed PERF-3577.

**Pull request merged**